packages/backend/src/config/parseConfig.ts

```diff
@@ -359,6 +388,13 @@ const mergeWithEnvironment = (config: LightdashConfigIn): LightdashConfig => {
                 oauth2TenantId: process.env.AUTH_AZURE_AD_OAUTH_TENANT_ID,
                 callbackPath: '/oauth/redirect/azuread',
                 loginPath: '/login/azuread',
+                x509PublicKeyCertPath: process.env.AUTH_AZURE_AD_X509_CERT_PATH,
```

Comment:
We should update our azure docs with the new env vars
- https://docs.lightdash.com/self-host/customize-deployment/use-sso-login-for-self-hosted-lightdash#azure-active-directory
- https://docs.lightdash.com/self-host/customize-deployment/environment-variables

Reply:
Done - there's some additional work here as far as outlining the `private_key_jwt` flow setup, but will hold off on that until we've had more time to test and gather any points to incorporate into the docs.